src/firefly/js/core/ReduxFlux.js

```diff
@@ -160,6 +163,16 @@ actionCreators.set('exampleDialog', (rawAction) => {
     return rawAction;
 });
 
+actionCreators.set('fileuploadDialog', (rawAction) => {
+    showFileUploadDialog();
+    return rawAction;
+});
+
+actionCreators.set('downloadDialog', (rawAction) => {
+    DownloadOptionPanel();
```

Comment:
showExampleDialog was done early on and was not cleaned up properly.  You should not have to register these dialogs as an action creator.  The action is SHOW_DIALOG in ComponentCntlr.  The added lines in this file should be removed.